index.js

```diff
@@ -33,9 +35,11 @@ module.exports = postcss.plugin(
         ? path.dirname(result.opts.to)
         : from
 
+      var cb = getDeclProcessor(result, from, to, callback, options, isCustom)
```

Comment:
Next, I create a closure function to save parameters, which are equal for all declarations.